heartbeat/monitors/monitor.go

```diff
@@ -92,13 +86,31 @@ func (e ErrDuplicateMonitorID) Error() string {
 	return fmt.Sprintf("monitor ID %s is configured for multiple monitors! IDs must be unique values.", e.ID)
 }
 
+// Creates a new monitor, without leaking resources in the event of an error.
```

Comment:
I wonder why hound did not complain here.

Reply:
My guess is because it isn't public. Fixed!